mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -160,22 +170,33 @@ object Config {
   case class Checker(
       tuning: Option[Map[String, String]] = Some(Map()),
       algo: Option[String] = Some("incremental"), // TODO: convert to case class
-      config: Option[String] = None,
+      config: Option[File] = None,
       discardDisabled: Option[Boolean] = Some(true),
       cinit: Option[String] = None,
-      init: Option[String] = None, // TODO Set proper default here once ConfigurationPassImpl is fixed
-      inv: Option[String] = None, // TODO Should be list?
-      next: Option[String] = None, // TODO Set proper default here once ConfigurationPassImpl is fixed
+      init: Option[String] = None,
+      inv: Option[List[String]] = None,
+      next: Option[String] = None,
       length: Option[Int] = Some(10),
       maxError: Option[Int] = Some(1),
       noDeadlocks: Option[Boolean] = Some(false),
       nworkers: Option[Int] = Some(1),
       smtEncoding: Option[SMTEncoding] = Some(SMTEncoding.OOPSLA19),
-      temporal: Option[String] = None, // TODO SHould be list?
+      temporal: Option[List[String]] = None,
```

Comment:
`temporal` sounds like it's a boolean, should this be named `temporalProperties` or something?

Reply:
Yes, i think that makes sense. I started from just picking up whatever names had been used in the option map, but I agree that's clearer.